packages/app/src/pages/PoolPage/AddLiquidity.tsx

```diff
@@ -230,6 +231,28 @@ export default function AddLiquidity() {
 
   const errorsCreatePull = validateCreatePool();
```

Comment:
I think here you can move this into `useEffect` as I mentioned that another pull request